ui.frontend/src/main/webpack/components/_teasercomponent.scss

```diff
@@ -0,0 +1,223 @@
+.cmp-teasercomponent {
+    position: relative;
+    width: $max-width;
+    height: $max-height;
+
+    &__bg {
+        min-height: 500px;
+        background-size: cover;
+        background-position: center;
+
+        @include min-screen(md) {
+            background-repeat: no-repeat;
+            background-size: cover;
+            height: $max-height;
+            min-height: 600px;
+            position: relative;
+            background-position: center;
+        }
+    }
+
+    &__container {
+        padding: 13px;
+        width: 70%;
+        background-color: $color-background;
+        bottom: 150px;
+        position: absolute;
+        right: 30px;
+
+
+        @include min-screen(md) {
+            background-color: $color-background;
+            position: static;
+            width: 35%;
+            padding: 10px;
+            position: absolute;
+            bottom: 150px;
+            right: 50px;
+            height: 33%;
+        }
+
+
+        &-text {
+            font-size: $font-size-md;
+            margin-bottom: 20px;
+            text-align: center;
+
+
+        }
+
+        &-pretitle {
+
+            margin-bottom: 20px;
+            font-size: $font-size-xsm;
+            text-align: center;
+        }
+
+        &-item {
+            text-align: center;
+            letter-spacing: $letter-spacing;
+            font-size: $font-size-sm;
+            display: flex;
+            width: 200px;
+            margin: 0 auto;
+
+
+            &-button {
+                background-color: $color-background-green;
+                display: flex;
+                flex-direction: column;
+                margin-right: 10px;
+                width: 100px;
```

Comment:
Won't fixing a button width create issue when user enter a text in dialog which span more than 100px